exporter/dynatraceexporter/serialization/serialization_test.go

```diff
@@ -28,10 +28,13 @@ func TestSerializeIntDataPoints(t *testing.T) {
 	}
 
 	intSlice := pdata.NewIntDataPointSlice()
-	intSlice.Resize(1)
+	intSlice.Resize(2)
 	intPoint := intSlice.At(0)
 	intPoint.SetValue(13)
 	intPoint.SetTimestamp(pdata.Timestamp(100_000_000))
+	intPoint1 := intSlice.At(1)
+	intPoint1.SetValue(13)
+	intPoint1.SetTimestamp(pdata.Timestamp(100_000_000))
```

Comment:
```suggestion
	intPoint0 := intSlice.At(0)
	intPoint0.SetValue(13)
	intPoint0.SetTimestamp(pdata.Timestamp(100_000_000))
	intPoint1 := intSlice.At(1)
	intPoint1.SetValue(14)
	intPoint1.SetTimestamp(pdata.Timestamp(101_000_000))
```
You could use different values to ensure nothing is mixed up or taken twice.